src/lambda/wizards/samDeployWizard.ts

```diff
@@ -95,110 +91,32 @@ export class SamDeployWizard extends MultiStepWizard<SamDeployWizardArgs> {
             ignoreFocusOut: true,
 
             // https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-s3-bucket-naming-requirements.html
```

Comment:
Move the reference urls for both validation methods down to the methods as well (they will get overlooked up here)

Reply:
✔️